lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/EntityBulkGetRequest.json

```diff
@@ -0,0 +1,22 @@
+{
+    "title":"EntityBulkGetRequest",
+    "description":"JSON schema for EntityBulkGetRequest object",
+    "implements":[
+        {
+            "$ref":"org.sagebionetworks.repo.model.asynch.AsynchronousRequestBody"
+        },
+		{
+			"$ref": "org.sagebionetworks.repo.model.table.HasEntityId"
+		}
+    ],
+    "properties":{
+        "entityIds":{
+            "type":"array",
+            "description": "The entity targets of this EntityBulkGetRequest",
+			"uniqueItems": false,
```

Comment:
need an array of objects that have: entityId, version number and mask (just like the getEntityBundle()).